src/liboslexec/backendllvm.h

```diff
@@ -279,6 +279,30 @@ class BackendLLVM final : public OSOProcessorBase {
     llvm::Type* llvm_type_closure_component();
     llvm::Type* llvm_type_closure_component_ptr();
 
+    llvm::Type* llvm_type_texture_options();
+    llvm::Type* llvm_type_texture_options_ptr();
+    llvm::Value* texture_options_ptr();
```

Comment:
Let's match naming convention from batched_backend for temporary stack variables and prefix with "temp_"